CHANGELOG.md

```diff
@@ -44,6 +44,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - **CUMULUS-2255**
   - Updated Terraform deployment code syntax for compatibility with version 0.13.6
+- **CUMULUS-2376**
```

Comment:
Let's also add a changelog entry under "Added" explaining that `post-to-cmr` now accepts this optional revision ID as part of its task input and will use it when publishing metadata to CMR